install_ads

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# USAGE
+# > ./install_ads
+#
+# This is the AxonDeepSeg (ADS) installer.
+#
+# If you do not want the crash reports question to be ask,
+# set ASK_REPORT_QUESTION at installation time like this:
+# >>> ASK_REPORT_QUESTION=false ./install_ads
+
+STATS_DSN="https://www.neuro.polymtl.ca/_media/downloads/stats_dsn_axondeepseg.txt"
```

Comment:
Yeah I'm getting rid of that download in https://github.com/neuropoly/spinalcordtoolbox/commit/32bdbd3477afb8e007cce9f37bbf8c3bd8d2bf97 ...

Reply:
6d24bcf